oracle-linux-image-tools/distr/ol8-slim/image-scripts.sh

```diff
@@ -41,6 +42,11 @@ distr::validate() {
 distr::kickstart() {
   local ks_file="$1"
 
+  local btrfs="\
+part btrfs.01 --fstype=\"btrfs\"  --ondisk=sda --size=4096 --grow\n\
+btrfs none --label=btr_pool --data=single btrfs.01\n\
+btrfs /    --subvol --name=root btr_pool\
+"
```

Comment:
We should match the partitioning table used by Anaconda when `btrfs` is selected, add a subvolume  for `/home` as well:

```
btrfs /home --subvol --name=home btr_pool
```